setup.cfg

```diff
@@ -9,6 +9,11 @@ ignore = F999,E203,E501,E128,E124,E402,W503,W504,E731,C901,B007,B009,B010,B011
 # We already have a lot of E501's - these are lines black didn't wrap.
 # But rather than append # noqa: E501 to all of them, we just ignore E501 for now.
 
+[flake8:local-plugins]
+paths = .
```

Comment:
i think it would be more consistent with the rest of the repo to create `src/sentry_flake8/` over making `script/` a python module

Reply:
my thought was that we didn't want/need to distribute this with the sentry pypi package so `src/` didn't make too much sense.  I didn't see another place to put this that wouldn't get packaged 🤔

Reply:
i see, i personally would prefer `scripts/flake8_plugin` and `paths = scripts/` then (to get rid of the generic-sounding top-level modules like `scripts`), but no strong preference